### PR TITLE
Add collaborative git

### PR DESCRIPTION
**Pull request by lyashevska**
## About
This PR moves Collaborative Version Control episode into git-lesson
 
### Changes
- Added a [markdown](https://github.com/esciencecenter-digital-skills/git-lesson/blob/37-move-collab-git-to-workbench/episodes/03-collaborative-git.md) file with teaching content.
- Added `03-collaborative-git.md` to `config.yaml`

### Requirements
See carpentries [workbench](https://carpentries.github.io/sandpaper-docs/) requirements

### Instructions
Follow steps outlined in the file.

## Related issues
- https://github.com/esciencecenter-digital-skills/good-practices-in-research-software-development/issues/37

### CODE REVIEW

File: episodes/03-collaborative-git.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+
+---
+title: Collaborative Version Control
+teaching: 180
+exercises: 2
```
Comment (svenvanderburg) [inline episodes/03-collaborative-git.md:5]: This should be minutes that it takes to solve exercises, not the number of exercises ;)

File: episodes/03-collaborative-git.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+::::::::::::::::::::::::::::::::::::::: objectives
+
+- Understand the basics of collaborative version control with git and Github
+- Understand the difference between centralized and distributed workflows
+
+::::::::::::::::::::::::::::::::::::::::::::::::::
+
+### Instructor notes
+Teaching is done as a pair of instructors. 
+Instructor A acts as the owner of the repository, instructor B as a collaborator (internal or external).
```
Comment (svenvanderburg) [inline episodes/03-collaborative-git.md:24]: Use the instructor note box, like in here: https://github.com/carpentries-incubator/deep-learning-intro/blob/7687adfb24a93e7dd2218ce01dd4884320cbf2ff/episodes/4-advanced-layer-types.Rmd#L300

This creates a box that only shows in instructor view.

**Review by svenvanderburg — COMMENTED** [review]
Looks good! I left a few comments. At this point it is not really useful as standalone material, but I don't think that is the aim here.

I would suggest to split this into 2 episodes: one on centralized workflow and one on distributed workflow. So they match a bit more in terms of length and cognitive load the 7 episodes in the SWC git intro part.

Feel free to do with my feedback what you want and then merge!

File: episodes/03-collaborative-git.md (hunk 1 above)
Comment (lyashevska) [reply episodes/03-collaborative-git.md:5]: true, 2 min is a bit short ;)

File: episodes/03-collaborative-git.md (hunk 2 above)
Comment (lyashevska) [reply episodes/03-collaborative-git.md:24]: Nice

**Comment by lyashevska** [comment]
Great, thanks for your feedback. I guess the solution part should be left empty.

**Comment by lyashevska** [comment]
This closes https://github.com/esciencecenter-digital-skills/good-practices-in-research-software-development/issues/37

**Pull request merged**